### DISCUSSION TITLE
flet-based desktop app: looking for Python-savvy alpha testers

### DISCUSSION THREAD

**Discussion by clstaudt** [flet-dev/flet#906] [Show and tell]
The Tuttle project is developing a free (as in beer and freedom) desktop app to help freelancers with managing projects, time and money.

https://github.com/tuttle-dev/tuttle/tree/dev

We decided to run with flet even though the framework is in an early stage. The last project phase was focused on UI.

<img width="1010" alt="Screenshot 2023-01-19 at 10 52 59" src="https://user-images.githubusercontent.com/875194/213456260-9414a0fb-0bbd-48df-9545-13ee6d3f030c.png">

We are are working on a release of version 1.0 and are looking for Python-savvy volunteers for an alpha test of the app. **Please reply with your GitHub handle if you are interested in running the demo and giving us some written feedback on GitHub**, for example:

- test installing the Python package and running the app
- test UI & UX
- code review 
- user feedback, especially if you are a freelancer

**Comment by ndonkoHenri** [comment]
@ndonkoHenri 🙂

**Reply by FeodorFitsner** [reply]
@FeodorFitsner

**Reply by clstaudt** [reply]
Welcome @ndonkoHenri & @FeodorFitsner. You have an invite to this team: https://github.com/orgs/tuttle-dev/teams/alpha-testers

**Reply by ndonkoHenri** [reply]
![image](https://user-images.githubusercontent.com/98978078/213603481-19660e59-b29c-4616-ad17-0ad1aad65551.png)

Error 404 on my end.

**Reply by clstaudt** [reply]
@ndonkoHenri Probably because you have not accepted the invite yet.

**Reply by iron3oxide** [reply]
@iron3oxide

**Reply by clstaudt** [reply]
Welcome @iron3oxide, you got an invite

**Reply by ndonkoHenri** [reply]
Where can I see the invite 😅

**Reply by clstaudt** [reply]
@ndonkoHenri  I don't know, I would expect it's in your notifications. Others have gotten it and were able to accept it.
